draft-ietf-quic-recovery.md

```diff
@@ -543,49 +544,46 @@ delay sending an acknowledgement.
 The PTO value MUST be set to at least kGranularity, to avoid the timer expiring
 immediately.
 
-When a PTO timer expires, the sender probes the network as described in the next
-section. The PTO period MUST be set to twice its current value. This exponential
-reduction in the sender's rate is important because the PTOs might be caused by
-loss of packets or acknowledgements due to severe congestion.
+When a PTO timer expires, the PTO period MUST be set to twice its current
+value. This exponential reduction in the sender's rate is important because
+the PTOs might be caused by loss of packets or acknowledgements due to severe
+congestion.
 
```

Comment:
Would we want some upper limit or this? What happens if idle timeout is not set - just indefinite inactivity?

Reply:
Why would there be no idle timeout?

Reply:
There should be, and maybe that is the answer. But I think a change was made to allow idle timeout to be indefinite. So I wonder how that will affect PTO.

Reply:
I don't remember any change like that. Every peer decides on its idle timeout. So even if a peer decides to have an infinite idle timeout (why would you do that to yourself?), once you reach a PTO duration on the order of what you might consider a typical idle timeout (let's say 30 seconds), exponential backoff should take care of your concern.

Reply:
Thats what I am asking - should the PTO just grow, or would you clamp it to 30 seconds or something?

Reply:
TCP clamps to a value(typically 30 seconds), but I think it's better to recommend a dead path timeout, such as a limit on the number of successive PTOs(I think some TCP impls default to 7?).

Reply:
Maybe it's worth adding a sentence here to say that "The life of a connection that is experiencing consecutive PTOs is limited by the endpoint's idle timeout."